src/blocks/CMakeLists.txt

```diff
@@ -45,4 +45,4 @@ foreach(BLOB_NAME checkpoints testnet_blocks stagenet_blocks)
     )
 endforeach()
 
-add_library(blocks STATIC blocks.cpp ${GENERATED_SOURCES})
```

Comment:
Was there a reason why blocks had to be STATIC?

Reply:
If the symbols can be read just fine, I don't think it needs to be static.